pipeline-examples/build-in-docker-container/BuildInDockerContainer.groovy

```diff
@@ -0,0 +1,18 @@
+node {
+    stage('checkout') {
+        git url: 'https://repository.com/foo/golang-project.git'
+    }
+    stage("build") {
+        writeFile file: "test.txt", text: "test"
+        docker.withRegistry("https://my-private-registry.intranet.net", 'docker-registry-credentials') {
+            docker.image("my-private-registry.intranet.net/tool/golang").inside("-v /mount_something_from_host:/mounted") { c ->
+                sh 'cat /mounted' // we can mount any file from host
```

Comment:
`cat`? For a volume?

Reply:
Hello @daniel-beck ,
yes we can mount any file from host and cat it - when you mount file in docker as volume you can do any read/write action on this file. Anyway I agree with you that it could be confusing so I've changed paths to less confusing.

Reply:
what is the use of this **c->** is it mandatory to add? I am new to this area. so trying to explore this. Would you plz explain the use of this **c->**